src/haproxy.py

```diff
@@ -133,6 +134,8 @@ def reconcile_ingress(
             "config_external_hostname": external_hostname,
             "haproxy_crt_dir": HAPROXY_CERTS_DIR,
         }
+        if HAPROXY_CAS_FILE.exists():
+            template_context["haproxy_cas_file"] = HAPROXY_CAS_FILE
```

Comment:
should we remove these lines since we're not touching the haproxy config for this PR?